be-traveltrove/src/main/java/com/traveltrove/betraveltrove/business/tour/TourService.java

```diff
@@ -8,4 +8,7 @@ public interface TourService {
 
     Flux<TourResponseModel> getTours();
     Mono<TourResponseModel> getTourByTourId(String tourId);
+    Mono<TourResponseModel> addTour(TourResponseModel tourResponseModel);
+    Mono<TourResponseModel> updateTour(String tourId, TourResponseModel tourResponseModel);
```

Comment:
I think these should take in TourRequestModels.

Reply:
Yes, i updated it after my PR. Thanks though!